src/components/Modal/src/Modal.vue

```diff
@@ -7,6 +7,37 @@
 	</div>
 </template>
 
+<script>
+import modalApi from './modal-api';
+
+export default {
+	name: 'Modal',
+
+	inject: {
+		modalApi,
+	},
+
+	props: {
+		/**
+		 * Before close hook, can block closing
+		 */
+		beforeClose: { // eslint-disable-line vue/require-default-prop
```

Comment:
@privatenumber I removed the default, but eslint was upset - I disabled this line but not sure if you would prefer me to do something else here

Reply:
Didn't think a default was necessary if the prop was not required.

You can explicitly set default to `undefined` instead.